### ISSUE TITLE
Update Figtree

### ISSUE THREAD

**Issue by emmamarichal** [google/fonts#8723]
**FONT:** FIGTREE

**Upstream Repo**
https://github.com/erikdkennedy/figtree

**Describe the bug**
Paths aren't merged and create rendering issues. See sub-issues related.

- https://github.com/google/fonts/issues/7380
- https://github.com/google/fonts/issues/7594

**Upgrade**
A PR is opened in the upstream repo: dy/figtree/pull/15
This updates the language support, adding cyrillic, but need to be reviewed.

### **To sum up:**

- [ ] merge paths to improve rendering
- [ ] review and add cyrillic

cc @erikdkennedy

**Comment by erikdkennedy** [comment]
@emmamarichal I just received another report that the unmerged paths are causing issues.

I noticed you opened this issue on yourself. Are you planning on merging the paths?

(It would be a huge help. I'm currently snuggling a newborn and don't have a ton of time in the near future 🙂)

**Comment by emmamarichal** [comment]
Will see what I can do, I didn't test it yet, but my colleague Simon gave me a tips to merge all the paths during the export process (to avoid changing sources). I'll try that! 

I add the command here to remember

`extraFontmakeArgs: --filter "ufo2ft.filters.decomposeComponents::DecomposeComponentsFilter"`

**Comment by Twilightssuperb** [comment]
How soon will Figtree be available on GF with Cyrillic glyphs, that seems to be just implemented in last releases?

**Comment by emmamarichal** [comment]
We need to review it carefully, since it's a quite big upgrade and a new language support. Also, there are now a lot of conflicts with the main branch, that have to be solved before the upgrade.
